tests/coverage_lcov/test_lcov.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+set -eu
+# for info see 'read.me'
+
+##############################################################
+# Folders structure (20.08.2020)
+# NOTE: Drawing will be removed after approval script
+##############################################################
+# facelift/                 <-- directory with source code
+# ├─ build/
+# │  ├─                     <-- working directory
+# │  └─
+# ├─ ci/
+# │  ├─ appveyor
+# │  └─ travis
+# ├─ cmake/
+# ├─ CMakeLists.txt
+# ├─ src/
+# ├─ tests/
+# │  ├─ asyncfunctions/
+# │  ├─ benchmarking/
+# │  ├─ cmake/
+# │  ├─ CMakeLists.txt
+# │  ├─ combined/
+# │  ├─ coverage_lcov/
+# │  │   ├─ llvm-gcov.sh    <--- patch for LCOV
+# │  │   ├─ read.me
+# │  │   └─ test_lcov.sh*   <--- executable script
```

Comment:
_-- NOTE: Drawing will be removed after approval script_

Note: this review will not get an approval with this comment and this note.

Reply:
Okay. I will remove this drawing.